packages/react/src/domain-config/ConfigView.tsx

```diff
@@ -4,13 +4,39 @@ import * as Lib from 'dm3-lib';
 import { ConnectionType } from '../reducers/Connection';
 import { ethers } from 'ethers';
 import StateButton, { ButtonState } from '../ui-shared/StateButton';
+import Icon from '../ui-shared/Icon';
 
 function ConfigView() {
     const [addrEnsName, setAddrEnsName] = useState<string | undefined>();
     const [dm3UserEnsName, setDm3UserEnsName] = useState<string | undefined>();
+    const [existingDm3UserEnsName, setExistingDm3UserEnsName] = useState<
+        string | undefined
+    >();
+
+    const [loading, setLoading] = useState<
```

Comment:
What is the purpose of this var?

Reply:
This var stores the name of the component which is loading.

Reply:
It is used to indicate loading and blocks the other button during loading

Reply:
I'd expect loading to be bool when it is solely used to indicate whether a component is loading. Is there a better naming for it?

Reply:
It can't be bool because it indicates what is loaded